.github/workflows/minion-tests.yml

```diff
@@ -16,32 +17,70 @@ jobs:
     steps:
       - uses: actions/checkout@v2
 
+      - name: Set up cache
+        uses: actions/cache@v3
+        with:
+          path: |
+            ~/.cargo/bin/
+            ~/.cargo/registry/index/
+            ~/.cargo/registry/cache/
+            ~/.cargo/git/db/
+            target/            
+            solvers/minion/vendor
+          key: ${{ runner.os }}-cargo-${{ hashFiles('**/Cargo.lock') }}
```

Comment:
the key probably should hash more files?

Reply:
Agreed.

I think I will need to hash it based on the commit hash for the minion submodule , so cache is invalidated when minion is changed?